circle.yml

```diff
@@ -614,30 +632,30 @@ workflows:
   testing:
     unless: <<pipeline.parameters.benchmark>>
     jobs:
-      - lint
-      - release-linux:
-          requires:
-            - release-native-linux
-      - release-native-linux
-      - release-macos:
-          requires:
-            - release-native-macos
-      - release-native-macos:
-          requires:
-            - sanitizers-macos
+#      - lint
```

Comment:
I will drop this commit before the merge. This PR only affects the "coverage-gcc" job.